installer/config.json.example

```diff
@@ -28,6 +28,7 @@
     "DNSSecurityFlags": 1,
     "DeadlockDetection": 0,
     "DeadlockDetectionThreshold": 30,
+    "DisableLedgerLRUCache": false,
```

Comment:
generated from `make generate`

Reply:
and now I discover `config/defaultsGenerator/defaultsGenerator.go` doesn't even use `encoding/json` nor `go-codec` but writes its own custom json text.

Reply:
oh you mean iterating through fields and use reflection to get type... yadiyada... and yes it seems to be some customized logic...

Reply:
workaround/fix
```
diff --git a/config/defaultsGenerator/defaultsGenerator.go b/config/defaultsGenerator/defaultsGenerator.go
index 9c3365f89..a03d008a5 100644
--- a/config/defaultsGenerator/defaultsGenerator.go
+++ b/config/defaultsGenerator/defaultsGenerator.go
@@ -112,6 +112,9 @@ func prettyPrint(c config.Local, format string) (out string) {
        }
 
        for fieldIdx, field := range fields {
+               if format == "json" && field.Tag.Get("codec") == "-" {
+                       continue
+               }
                switch field.Type.Kind() {
                case reflect.Bool:
                        v := reflect.ValueOf(&c).Elem().FieldByName(field.Name).Bool()
```

Reply:
^ was working on something similar on my local end